src/open_inwoner/accounts/views/profile.py

```diff
@@ -40,6 +41,24 @@
 logger = logging.getLogger(__name__)
 
 
+class ErrorMessageMixin(FormMixin):
```

Comment:
I agree with your comment that displaying the error message twice is overkill, so this class can be removed.